tests/test_core/test_sequence.py

```diff
@@ -66,12 +68,7 @@ def test_annotate_matches_to(self):
             name="fred",
             allow_multiple=True,
         )
-        regular_expression = DNA.to_regex(seq=pattern)
-        for i in range(0, len(annot)):
-            fred = annot[i].get_slice()[0:len(pattern)]
-            self.assertEqual(
-                str(fred), re.search(regular_expression, str(fred)).group()
-            )
+        self.assertEqual([a.get_slice() for a in annot], ["CCAC", "CCGC"])
         annot = seq.annotate_matches_to(
             pattern=pattern,
             annot_type="domain",
```

Comment:
you need to add the test for an ascii sequence returning `[]`

Reply:
ASCII test is performed lines 81 through 88